### PR TITLE
Remove extraneous environmental parameters

### PR DESCRIPTION
**Pull request by dbernstein**
Resolves: https://jira.lyrasis.org/browse/FCREPO-3437

Follow the updated instructions in the README to test.  You can now pass your fcrepo configuration as a single properties file or via the CATALINA_OPTS environment property (or both).   

This PR removes the previous and overly complicated approach that  used myriad environment variables.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,41 +32,47 @@ Variable | Default Value | Description
...
-`FCREPO_ACTIVEMQ_CONFIGURATION` | none | Specify a custom fcrepo.activemq.configuration
-`FCREPO_ACTIVEMQ_DIRECTORY` | `/var/lib/fcrepo/activemq` | Set fcrepo.activemq.directory
-`FCREPO_AUTH_WEBAC_AUTHORIZATION` | none | Specify a custom root WebAC authentication fcrepo.activemq.configuration
-`FCREPO_SPRING_AUDIT_CONFIGURATION` | none | Specify a custom fcrepo.spring.audit.configuration
-`FCREPO_VELOCITY_RUNTIME_LOG` | none | Set fcrepo.velocity.runtime.log
 `FEDORA_ADMIN_USERNAME` | `fedoraAdmin` | If using the default tomcat-users.xml file: specify a custom username for the user `fedoraAdmin`
 `FEDORA_ADMIN_PASSWORD` | `fedoraAdmin` | If using the default tomcat-users.xml file: specify a custom password  for the FEDORA_ADMIN_USERNAME defined above
 `LOGBACK_CONFIGURATIONFILE` | none | Specify a custom logback.configurationFile
 
+Use the CATALINA_OPTS environment variable to pass in Java options (-Dname=value). You may also pass all your options via a properties vile
```
Comment (awoods) [inline README.md]: typo: `vile` -> `file`

File: README.md [hunk 2]
```diff
@@ -32,41 +32,47 @@ Variable | Default Value | Description
...
+
 For a detailed explanation of the configuration options have a look at [Application Configuration](https://wiki.lyrasis.org/display/FEDORA6x/Application+Configuration) in the Lyrasis Wiki.
 
 ## Development
 
-### Repository organisation
+### Repository organization
 
-This repository is used to manage docker images for the Fedora Commons from Version 5.x onwards. The Dockerfile for the latest release of each major version is managed on separate branch. Currently the following are planned:
+This repository is used to manage docker images for the Fedora Commons from Version 5.x onwards. The Dockerfile for the latest release of each major version is managed on separate branch.
```
Comment (awoods) [inline README.md]: typo: "..on [a? its?] separate branch."

**Review by awoods — CHANGES_REQUESTED** [review]

**Comment by awoods** [comment]
I likely have something incorrect, but the following command did not change the `fcrepo.home` directory from `/var/lib/fcrepo`.
```
docker run -d -p8080:8080 -v/tmp/awoods/data:/fedora_home -v/tmp/awoods/data/fcrepo.properties:/fcrepo.properties -e CATALINA_OPTS="-Dfcrepo.config.file=/fcrepo.properties" --name=fcrepo fcrepo/fcrepo
```
..where the contents of "fcrepo.properties" is:
```
fcrepo.home=/fedora_home
```

**Comment by dbernstein** [comment]
@awoods :  try replacing 
```
-v/tmp/awoods/data/fcrepo.properties:/fcrepo.properties
```
with 
```
-v/tmp/awoods/fcrepo.properties:/fcrepo.properties
```
and move your fcrepo.properties file to /tmp/awoods/fcrepo.properties

I think the problem is mounting a file within a mounted directory.

**Review by awoods — APPROVED** [review]
Works now. I needed to build the container.

**Pull request merged**